### PR TITLE
testing/py-avro: new aport

### PR DESCRIPTION
**Pull request by gena01**

### CODE REVIEW

**Comment by maxice8** [comment]
pkgname= is py3-avro but directory it resides is py-avro

File: testing/py-avro/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+arch="noarch"
+license="Apache-2.0"
+depends="py3-six"
+makedepends="py3-setuptools python3-dev"
+source="$pkgname-$pkgver.tar.gz::http://apache.mirrors.tds.net/avro/stable/py3/$_pkgname-$pkgver.tar.gz"
+builddir="$srcdir/$_pkgname-$pkgver"
+
+build() {
+	python3 setup.py build
+}
```
Comment (maxice8) [inline testing/py-avro/APKBUILD]: missing check()

Comment (gena01) [reply testing/py-avro/APKBUILD]: There are no tests? in the archive file?

Comment (gena01) [reply testing/py-avro/APKBUILD]: Logged a ticket: https://issues.apache.org/jira/browse/AVRO-2420

**Comment by maxice8** [comment]
commit title is wrong, it is py3-avro

**Pull request closed without merging**